test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java

```diff
@@ -228,6 +231,90 @@ public static String getExecutableDesciption(Path pathToExeFile) {
                 "Failed to get file description of [%s]", pathToExeFile));
     }
 
+    public static void killProcess(long pid) {
+        Executor.of("taskkill", "/F", "/PID", Long.toString(pid)).dumpOutput(true).execute();
```

Comment:
In case of two processes (parent and child) it will terminate only parent. Based on `taskkill /?` `/T` needs to be specified to terminate child process as well or call it for both processes.

Reply:
It is sufficient to terminate the parent process only. The child process will be terminated automatically. Automatic app launcher child process termination was implemented in [JDK-8301247](https://bugs.openjdk.org/browse/JDK-8301247)